content/en/logs/log_collection/_index.md

```diff
@@ -24,34 +24,41 @@ further_reading:
+## Overview
 
-Datadog Log Management also comes with a set of out of the box solutions to collect your logs and send them to Datadog:
+Choose a configuration option below to begin ingesting your logs. If you are already using a log-shipper daemon, refer to the dedicated documentation for [Rsyslog][1], [Syslog-ng][2], [NXlog][3], [FluentD][4], and [Logstash][5].
 
-* [**Collect logs from your hosts**][2].
-* [**Collect logs from your applications**](?tab=ussite#application-log-collection).
-* [**Collect logs from a Docker environment**](?tab=ussite#container-log-collection).
-* [**Collect logs from a serverless environment**](?tab=ussite#serverless-log-collection).
-* [**Collect logs from your Cloud provider**](?tab=ussite#cloud-providers-log-collection).
+There is also a [list of available Datadog Log collection endpoints](#datadog-logs-endpoints) if you want to send your logs directly to Datadog.
 
-Datadog Integrations and Log Collection are tied together. Use an integration default configuration file to enable its dedicated [processing][10], [parsing][11], and [facets][12] in Datadog.
+**Note**: When sending logs in a JSON format to Datadog, there is a set of reserved attributes that have a specific meaning within Datadog. See the [Reserved Attributes section](#reserved-attributes) to learn more.
 
-<div class="alert alert-warning">
-<a href="/integrations/#cat-log-collection">Consult the current list of available supported integrations</a>.
-</div>
+## Setup
 
-Find at the bottom of this page the [list of available Datadog Log collection endpoints](#datadog-logs-endpoints) if you want to send your logs directly to Datadog.
+{{< tabs >}}
+{{% tab "Host" %}}
 
-**Note**: When sending logs in a JSON format to Datadog, there is a set of reserved attributes that have a specific meaning within Datadog. See the [Reserved Attributes section](#reserved-attributes) to learn more.
+Follow the [Datadog Agent installation instructions][1] to start forwarding logs alongside your metrics and traces. The Agent can [tail log files][2] or [listen for logs sent over UDP / TCP][2], and you can configure it to [filter out logs][3], [scrub sensitive data][3], or  aggregate [multi line logs][4].
+
+[1]: /agent/logs/
+[2]: /agent/logs/#custom-log-collection
+[3]: /agent/logs/advanced_log_collection/#filter-logs
+[4]: /agent/logs/advanced_log_collection/#multi-line-aggregation
+
+{{% /tab %}}
 
-## Application Log collection
+{{% tab "Application" %}}
 
 After you have [enabled log collection][1], configure your application language to generate logs:
 
 {{< partial name="logs/logs-languages.html" >}}
 
-## Container Log Collection
+**Note**: JSON-formatted logging helps handle multi-line application logs, and is automatically parsed by Datadog. If you have control over the log format you send to Datadog, it is recommended to format logs as JSON to avoid the need for custom parsing rules.
```

Comment:
```suggestion
**Note**: JSON-formatted logging helps handle multi-line application logs. JSON-formatted logs are automatically parsed by Datadog. If you have control over the log format you send to Datadog, it is recommended that you format logs as JSON to avoid the need for custom parsing rules.
```